pkg/operator/context/apis.go

```diff
@@ -20,13 +20,11 @@ import (
 	"bytes"
 	"strings"
 
-	"github.com/cortexlabs/cortex/pkg/lib/errors"
 	"github.com/cortexlabs/cortex/pkg/lib/hash"
 	s "github.com/cortexlabs/cortex/pkg/lib/strings"
 	"github.com/cortexlabs/cortex/pkg/operator/api/context"
 	"github.com/cortexlabs/cortex/pkg/operator/api/resource"
 	"github.com/cortexlabs/cortex/pkg/operator/api/userconfig"
-	"github.com/cortexlabs/cortex/pkg/operator/config"
 )
 
 func getAPIs(config *userconfig.Config,
```

Comment:
nit: we should either add a new line for the first parameter, or make the function signature all one line